### PR TITLE
Adjust IO tests for dask-expr migration

### PR DESCRIPTION
**Pull request by phofl**
- [ ] Closes #xxxx
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

File: dask/dataframe/io/tests/test_parquet.py [hunk 1]
```diff
@@ -598,6 +603,7 @@ def test_roundtrip_nullable_dtypes(tmp_path):
     assert_eq(df, ddf2)
 
 
+@pytest.mark.skipif(dd._dask_expr_enabled(), reason="not supported")
```
Comment (crusaderky) [inline dask/dataframe/io/tests/test_parquet.py:606]:
```suggestion
@pytest.mark.skipif(DASK_EXPR_ENABLED, reason="not supported")
```

File: dask/dataframe/tests/test_rolling.py [hunk 2]
```diff
@@ -574,6 +575,7 @@ def test_groupby_rolling():
     assert_eq(expected, actual, check_divisions=False)
 
 
+@pytest.mark.xfail(dd._dask_expr_enabled(), reason="this works in dask-expr")
```
Comment (crusaderky) [inline dask/dataframe/tests/test_rolling.py:578]: Not sure I understand the `reason` here?

Comment (phofl) [reply dask/dataframe/tests/test_rolling.py:578]: Dask raises but we made this work in dask-expr, the tests checks for a ValueError, but dask-expr just completes without raising, so the test will fail

**Comment by phofl** [comment]
Going ahead and merging this if no further comments

**Review by crusaderky — APPROVED** [review]

**Pull request merged**